assets/src/block-editor/plugins/amp-toggle.js

```diff
@@ -60,6 +61,13 @@ function AMPToggle( { isEnabled, onChange } ) {
 					)
 				}
 			</PluginPostStatusInfo>
+			{
+				isEnabled && (
+					<PluginPostStatusInfo>
+						<AMPPreview />
```

Comment:
This should only be displayed of the template mode is not Standard. It's only applicable to Transitional and Reader modes.

Reply:
Ah, right.

Reply:
This now only displays if:

```js
isEnabled && ! isStandardMode
```